### ISSUE TITLE
[Feature] Burger Shot

### ISSUE THREAD

**Issue by jeebbers** [FrenzyRoleplay/server-issues#649]
A clear and concise description of what the problem is.
**As much as I love Frenzy I have noticed it is hugely favorited to gang RP. It lacks places around the city where any body can enter the building and socialize without being worried about being shot or running into some sort of conflict. Even at Pillbox the violent crimes happening there have been an issue. I understand not everybody will get along IC but it would be nice to have a place where you can go to socialize and give the opportunity for civs to create different RP scenes that isn't gang related or criminal rp.**

I would like to get Burger Shot built. As the owner of Burger Shot my goal would be to create a fun environment for everybody to grab food and socialize. Adding the drive thru feature would add to the RP and being able to grab deliveries through the drive-thru would be a cool little feature as well. My goal would be to set up events sponsored by Burger Shot to create some more community activities either small or big. Example - put together some kind of food eating contest. Adding the job of a mascot for Burger Shot gives the opportunity for someone to create a unique character.  I also had an idea (as seen in another city) that with a specific meal you receive something at random. Whether we start a collectible item around the city like a toy, a player card (sports? people around the city?), or even snow globes and then also have a very very very rare chance of receiving a weed seed or something of that sorts at times. This would help with bringing in a consist amount of people into Burger Shot. 

**Describe alternatives you've considered**
If Burger Shot is not possible to build we have thought about opening Up n' Atom or a Cluckin' Bell as an alternative. 
**Additional context**

I wrote out an SOP already and a menu. I can send those over via DM's if requested. I just don't want it out there for everybody to see to help keep the element of surprise with what Burger Shot would look like! 
If this MLO doesn't work, we have more we can send. We have no been able to find one on putting a drive thru in. If the drive thru is not possible I guess we can make do without it.... :(  
https://www.gta5-mods.com/maps/gtaiv-burgershot-interior-sp-and-fivem

**Comment by nixety** [comment]
I agree and am working on bringing interiors around the city of various sizes for civs to run business's from.
I like that MLO. But I like to be different, especially setting frenzy apart from NoPixel, So Im in the process of building a burger business in the up 'n' atom burger place along vinewood blvd. once that is done we will look at your SOP and go from there...interiors take so much time to build so please bear with us.

**Comment by jeebbers** [comment]
Hi Nixety! I was wondering if there was an update on the burger business?

**Comment by nixety** [comment]
It is on the road-map to be developed, we will be looking into potential fps/texture issues with the various MLO's soon.
We will announce when we are ready to roll these out. A working, stable & playable server is our focus.
